### PR TITLE
[MERGE] website_form: add some usability imp to the form builder

### PR DESCRIPTION
**Pull request by dbh-odoo**
Note: the form builder will be refactored in a deeper way in a next update.
See sub-commits for details.

task-1874350

Thanks to @dbh-odoo @pna-odoo and @dja-odoo for the work on this task.

### CODE REVIEW

**Comment by dja-odoo** [comment]
LGTM

File: addons/website_form/static/src/js/website_form_editor.js [hunk 1]
```diff
@@ -458,11 +544,16 @@ odoo.define('website_form_editor', function (require) {
...
                 // Reconstruct the field from the select tag
                 var select = this.$target.find('select');
+                var $label = this.$target.find('label.col-form-label').parent();
                 var field = {
                     name: select.attr('name'),
                     string: this.$target.find('.col-form-label').text().trim(),
                     required: self.$target.hasClass('o_website_form_required'),
                     custom: self.$target.hasClass('o_website_form_custom'),
+                    formatInfo: {
+                        lableClass: $label.attr('class') || '',
```
Comment (JKE-be) [inline addons/website_form/static/src/js/website_form_editor.js]: label

**Comment by qsm-odoo** [comment]
@robodoo rebase-merge r+

**Comment by qsm-odoo** [comment]
@robodoo rebase-merge r+ retry

**Pull request closed without merging**